Various Slack page embetterments (#1921)
writethedocs/www: docs/slack.rst

There is a specific set of behaviors that is prevalent in online communities that we'd like to address, to make sure our community doesn't fall into these traps. 
Please avoid these behaviors:

> - **No black sheep**. - There may be organizations, job functions, and tools that you don't like.

Reviewer: (very optional, style)

I like the improved consistency! I didn't do this in my last big PR against this because the diff was already big. If you wanted to go a step further toward consistency, they could all become:

- Complete sentences ending in a full stop
- In bold
- Followed by a semantic line break

That is:

```suggestion
- **No black sheep.**
  There may be organizations, job functions, and tools that you don't like.
```

  There is a good chance there is someone in the room who is closely connected to that thing you don't like.
  Don't make them feel bad; they are likely just in a different context.

> - **Let me Google that for you (LMGTFY)** - If people ask questions that are simple to answer on Google, still answer them nicely.

Reviewer: And to further illustrate my previous comment:

```suggestion
- **Let me Google that for you (LMGTFY).**
  If people ask questions that are simple to answer on Google, still answer them nicely.
```

  The Write the Docs Slack is informal and conversational; high-pressure sales aren't welcome.
  If people aren't interested in what you're offering, don't take it as a license to become louder and more aggressive.
  If you're asked to stop, then stop.
- **Treat the community like friends, not customers.**

>   Some posts can come off as infomercials, and those are harder for the community to engage with and for the moderation team to distinguish from spam.

Reviewer: (optional, style)

"can come off as" struck me a potentially fraught construction in this context, particularly for folks less fluent in English.

(Also inspired by my recent feeling of being in some sort of Greek myth: a man commits hubris by using too many phrasal verbs, so the gods condemn him to learn Dutch.)

```suggestion
  Posts that resemble infomercials are harder for the community to engage with and for the moderation team to distinguish from spam.
```

  If you post about things that aren't relevant to the documentation field or the channel you're posting in, moderators might delete your messages.

>   We want to give you the chance to talk about work you're excited about, but also want to make sure conversations stay relevant and not sales-focused.

Reviewer: (optional, style)

I like this a lot as-is, but my suggestion here is to make it more imperative, like the other entries in this list. I think we ought to tell people what to do to be successful, rather than intuit that from a statement of intent.

```suggestion
  Make sure conversations stay relevant and not sales-focused, so you can have the chance to talk about work you're excited about.
```
